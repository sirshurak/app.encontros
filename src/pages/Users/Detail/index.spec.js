import { shallow } from 'enzyme';
import UsersDetail from './index';

describe('UsersDetail component', () => {
    it('should match shallow snapshop', () => {
        const component = shallow(<UsersDetail />)
        expect(component).toMatchSnapshot();
    });
});