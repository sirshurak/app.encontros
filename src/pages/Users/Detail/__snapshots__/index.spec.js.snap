// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersDetail component should match shallow snapshop 1`] = `
<div
  className="users-detail"
>
  <UserDetail />
  <GoHome />
</div>
`;
